lib/ansible/modules/files/find.py

```diff
@@ -414,7 +414,6 @@ def main():
                     if files or dirs:
                         depth += 1
                     if depth > params['depth']:
-                        del(dirs[:])
```

Comment:
Note: I looked at this and looked at this and experimented on the REPL... I don't think this is actually a left-hand-side slice but it's not a right hand side slice either.  I think it's a special case that del handles.  Irregardless of that, it's also confusing what it is actually doing *and* apparently unnecessary here.  The line after this does the next iteration of the loop.  And the loop rebinds the dirs variable to a new list.  So the elements in this old version of the list will be available for garbage collecting immediately anyways.